contract/src/events.rs

```diff
@@ -0,0 +1,16 @@
+pub mod events_cep47;
+pub mod events_cep78;
+
+pub(crate) enum Event {
+    Cep47(events_cep47::CEP47Event),
+    Cep47Dict(events_cep47::CEP47Event),
+    Cep78,
+}
+
+pub(crate) fn record_event(event_enum: Event) {
+    match event_enum {
+        Event::Cep47(cep47_style) => events_cep47::record_event(&cep47_style),
+        Event::Cep47Dict(cep47_style) => events_cep47::record_event_dictionary(&cep47_style),
```

Comment:
I think I renamed `cep47_style` to `event` or it was `event` on first case, but I did not rename the second case. So it's not very consistent in my PR. https://github.com/casper-ecosystem/cep-78-enhanced-nft/pull/136/files#diff-e6b367605eb370c608fd74585147b016fc1e3b7fbd190772df677f70c77a3302R15

Can I rename it to `event` for the second case Cep47Dict to ?

thanks

Reply:
Sure, you can.